packages/cosmic-swingset/lib/ag-solo/web.js

```diff
@@ -140,22 +146,19 @@ export function makeHTTPListener(basedir, port, host, rawInboundCommand) {
     const connectionID = lastConnectionID;
     const id = `${req.socket.remoteAddress}:${req.socket.remotePort}[${connectionID}]:`;
 
-    console.log(id, `new ws connection ${req.url}`);
+    log.info(id, `new WebSocket connection ${req.url}`);
 
     ws.on('error', err => {
-      console.log(id, 'client error', err);
+      log.error(id, 'client error', err);
     });
 
     ws.on('close', (_code, _reason) => {
-      console.log(id, 'client closed');
+      log.info(id, 'client closed');
       broadcasts.delete(ws);
       if (req.url === '/captp') {
-        inboundCommand(
-          { type: 'CTP_CLOSE', connectionID },
-          req,
-          id,
-        ).catch(_ => {});
-        points.delete(connectionID);
+        inboundCommand({ type: 'CTP_CLOSE', connectionID }, req, id)
```

Comment:
This looks like an unrelated (but possibly correct) change to the ordering of `points.delete` and the command handler.. maybe put it in a separate cleanup PR? It's not immediately clear to me that we should defer removing it until after the command handler finishes.

Reply:
This happens after the `close` event, so the only reason to keep it around is for the `CTP_CLOSE` not to complain that the point doesn't exist.